db/migrate/20241008053852_add_external_voucher_id_external_voucher_set_id_voucher_type_to_vouchers.rb

```diff
@@ -0,0 +1,7 @@
+class AddExternalVoucherIdExternalVoucherSetIdVoucherTypeToVouchers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :vouchers, :external_voucher_id, :uuid
+    add_column :vouchers, :external_voucher_set_id, :uuid
```

Comment:
It looks like these are needed for redemption. But if we're able to validate with only the short code, then why do we need them to redeem? Do you know if there's a reason the VINE app couldn't just look up the short code? 

That would make our system a lot simpler.

Or if we do need to store these values, I'd suggest a shorter name without redundant "voucher" (ie `external_id`). Do we need to validate their uniqueness?
Or maybe we can simply store in a data field, if these values don't have any other meaning to the OFN app. Hmm or just ignore me and go with what you did already ;)

Reply:
> It looks like these are needed for redemption. But if we're able to validate with only the short code, then why do we need them to redeem? Do you know if there's a reason the VINE app couldn't just look up the short code?

I am not sure why it's designed this way, presumably when you redeem a voucher providing the voucher_id VINE doesn't need to do all the checks already done when validating a voucher.